src/dynamic-page-layout.scss

```diff
@@ -0,0 +1,361 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-dynamic-page-layout;
+
+.#{$block} {
+
+  $fd-dynamic-page-layout-background-color: var(--sapObjectHeader_Background);
+  $fd-dynamic-page-layout-content-background-color: var(--sapBackgroundColor);
+  $fd-dynamic-page-layout-padding-top-bottom: 0.5rem;
+  $fd-dynamic-page-layout-header-padding-top-bottom: 1rem;
+
+  @include fd-reset();
+
+  @mixin responsive-page($padding-y: 0,$padding-x: 1rem) {
+    padding: $padding-y $padding-x;
+  }
+
+  @mixin box-shadow() {
+    box-shadow: var(--sapContent_HeaderShadow);
+    // box-shadow: 0 0.25rem 0.25rem -0.25rem rgba(0,0,0,0.15), inset 0 -0.0625rem 0 0 var(--sapObjectHeader_BorderColor);
+  }
+
+  @mixin no-shadow {
+    &--no-shadow {
+      box-shadow: none;
+    }
+  }
+  // todo: check for mixin naming conventoin
+  @mixin flex-config($justify-content: space-between) {
+    display: flex;
+    flex-direction: row;
+    justify-content: $justify-content;
+    align-items: center;
+  }
+
+  @mixin custom-button() {
+    @include flex-config($justify-content: space-evenly);
+
+    margin: 0 0.25rem;
+    width: 1.5rem;
+    height: 2rem;
+    min-width: 1.5rem;
+  }
+
+  &__title-area {
+    @include fd-reset();
+
+    @include fd-flex($direction: column) {
+      justify-content: space-between;
+    }
+
+    min-height: 3rem;
+    word-wrap: break-word;
+    background-color: $fd-dynamic-page-layout-background-color; // discrepancy
+
+    @include fd-hover() {
+      background: var(--sapObjectHeader_Hover_Background);    // color not yet available in Fundamental, therefore hover not reflecting the color
+    }
+
+    &--s {
+      @include responsive-page($padding-y: $fd-dynamic-page-layout-padding-top-bottom, $padding-x: 0.5rem);
+    }
+
+    &--m_l {
+      @include responsive-page($padding-y: $fd-dynamic-page-layout-padding-top-bottom, $padding-x: 2rem);
+    }
+
+    &--xl {
+      @include responsive-page($padding-y: $fd-dynamic-page-layout-padding-top-bottom, $padding-x: 3rem);
+    }
+
+    &--transparent-bg {
+      background: transparent;  // WARNING: theme conflicting values
+    }
+
+    &--collapsed {
+      padding-bottom: 1rem;
+    }
+
+    &[aria-hidden="true"] {
+      display: none;
+    }
+  }
+
+  &__title-container {
+    @include fd-reset();
+    @include flex-config($justify-content: flex-start);
+  }
+
+  .#{$fd-namespace}-breadcrumb.#{$block}__breadcrumb {
+    @include fd-reset();
```

Comment:
You should not have the reset in this case. It will reset the styling coming from `.fd-breadcrumb`

Reply:
removed `fd-reset()`